Sources/OVP/KalturaOVPPlayer.swift

```diff
@@ -95,8 +97,10 @@ import PlayKitProviders
             return
         }
         
-        // Update KavaPlugin for specific Media
-        self.updateKavaPlugin(partnerId: ovpPartnerId, entryId: mediaEntry.id, mediaOptions: mediaOptions as? OVPMediaOptions)
+        if !self.isPluginLoaded(pluginName: KavaPlugin.pluginName) {
+            // Update KavaPlugin if it was not set explicitly for specific Media.
+            self.updateKavaPlugin(partnerId: ovpPartnerId, entryId: mediaEntry.id, mediaOptions: mediaOptions as? OVPMediaOptions)
+        }
```

Comment:
Please revert this, we should just call the updateKavaPlugin function as done in the OTTPlayer.
There it is checking if it is coming from the client or from us (defaultPluginConfigKeys).